qcodes/instrument_drivers/tektronix/Keithley_6500.py

```diff
@@ -171,7 +171,7 @@ def __init__(self, name, address, reset_device=False, **kwargs):
                            get_parser=float,
                            get_cmd='ROUT:SCAN:INT?',
                            set_cmd='ROUT:SCAN:INT {}',
-                           unit='s', vals=Numbers(min_value=0.001, max_value=999999.999))
```

Comment:
and what does the value of 0 mean in this case? trigger immediately? if so, i guess this is to be set using trigger_source parameter, no?

Reply:
trigger immediately I think. The problem here is that the default value from the device is 0, so if you read the value from the device and try to reapply it, it fails.

Reply:
oh.... ok, could you add that clarification as a comment next to that parameter definition? and then we'll merge.

Reply:
done